src/app/resources/skill.resource.js

```diff
@@ -0,0 +1,12 @@
+ (function() {
+    'use strict';
+
+    angular
+        .module('resource')
+        .factory('Skill', Skill);
+
+    function Skill(Resource, config) {
```

Comment:
I have a hard time settling on this myself, but I think I prefer plural resource names atm :smile: